packages/material-ui-unstyled/src/SliderUnstyled/SliderUnstyled.js

```diff
@@ -545,10 +580,12 @@ const SliderUnstyled = React.forwardRef(function SliderUnstyled(props, ref) {
 
   const trackOffset = valueToPercent(range ? values[0] : min, min, max);
   const trackLeap = valueToPercent(values[values.length - 1], min, max) - trackOffset;
-  const trackStyle = {
-    ...axisProps[axis].offset(trackOffset),
-    ...axisProps[axis].leap(trackLeap),
-  };
+
+  const sliderDimensions = sliderRef.current
+    ? sliderRef.current.getBoundingClientRect()
```

Comment:
This breaks server-side support:

<img width="278" alt="Capture d’écran 2021-02-15 à 15 23 54" src="https://user-images.githubusercontent.com/3165635/107958131-dd691000-6fa1-11eb-91cd-8e4da24d6e77.png">